lib/concurrent/atomic/condition.rb

```diff
@@ -66,5 +66,17 @@ def broadcast
       true
     end
 
+    private
+
+    if defined?(Process::CLOCK_MONOTONIC)
+      def clock_time
+        Process.clock_time Process::CLOCK_MONOTONIC
```

Comment:
@tenderlove Should this method be `clock_gettime`?

This is a Ruby feature I wasn't aware of until you created this PR so thank you for pointing it out. This solves a problem we've discussed in another issue. A ton of tests are failing, however. Should the method call be this? http://ruby-doc.org//core-2.2.0/Process.html#method-c-clock_gettime